components/metadata/metadata_traits.py

```diff
@@ -26,6 +26,9 @@ class FileMetadataTraits(MetadataTraitsBase):
     def __init__(self, a_object_value, a_display_name):
         MetadataTraitsBase.__init__(self, a_object_value, a_display_name)
 
+    # This function builds a bean that is posted to RDM for the purpose of representing a file.
+    # The _id field represents the file entity/object and hence will be the same for multiple revisions of the same file.
+    # This bean is not concerned with file upload identifiers. See FileUploadbean class in file_upload.py.

```

Comment:
Not sure what you mean by this, the bean contains the upload-uuid in the "uuid" field

Reply:
or the "sitelink_file_id" field

Reply:
Good find. Misplaced comment.